packages/block-library/src/navigation-overlay-close/block.json

```
@@ -0,0 +1,49 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/navigation-overlay-close",
+	"title": "Navigation Overlay Close",
+	"category": "design",
+	"description": "Add a Close button to your Navigation Overlay.",
+	"textdomain": "default",
+	"icon": "closeSmall",
+	"attributes": {
+		"hasIcon": {
+			"type": "boolean",
+			"default": true
+		}
+	},
+	"usesContext": [
+		"textColor",
+		"customTextColor",
+		"backgroundColor",
+		"customBackgroundColor"
+	],
+	"supports": {
+		"reusable": false,
```

```suggestion
		"reusable": false,
		"multiple": false,
```

I would also suggest only allowing this block to be inserted once

---

✅